Tests/RegressionTests.cs

```diff
@@ -150,21 +150,21 @@ private static TestCaseData[] GetRegressionTestParameters()
                 {"Total Trades", "1"},
                 {"Average Win", "0%"},
                 {"Average Loss", "0%"},
-                {"Compounding Annual Return", "155.210%"},
+                {"Compounding Annual Return", "191.086%"},
```

Comment:
Have opted for the overload option in 6a50816 and tried to harmonise with future symbolproperties changes. To minimize bloat of the API surface I have omitted some obvious method overloads but these can be added if preferred.